src/main/java/com/agilethought/atsceapi/controller/UserController.java

```diff
@@ -1,5 +1,24 @@
 package com.agilethought.atsceapi.controller;
 
-public class UserController {
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.http.HttpStatus;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.List;
 
-}
+import com.agilethought.atsceapi.service.UserService;
+import com.agilethought.atsceapi.model.UserModel;
+
+@RestController
+public class UserController {
+	@Autowired
+	private UserService service;
+	
+	@ResponseStatus(HttpStatus.OK)
+	@GetMapping("/at-sce-api/api/v1/users")

```

Comment:
Separate this annotation in two like base path for request mapping and resourse path. Example: 
`@RestController
@RequestMapping("/basePath/.../v1")
public class Controller {`

and for your method: 
`@GetMapping("/resource")
    @ResponseStatus(HttpStatus.OK)
    public Object getObjects ( ) {`